docs/installation-docker.rst

```diff
@@ -0,0 +1,81 @@
+=======================
+Installation via Docker
+=======================
+
+Kuma has experimental support for `Docker`_ as an alternative to the
+Vagrant set up.
+
+.. _Docker: https://www.docker.com/
+
+Docker setup
+============
+
+#. Ensure you have cloned the kuma Git repository and it is your current
+   working directory::
+
+        git clone git@github.com:mozilla/kuma.git
+        cd kuma
+
+#. Install the `Docker Toolbox`_. This will install the Docker engine,
+   along with Docker Compose and Docker Machine.
+
+   .. _Docker Toolbox: https://www.docker.com/products/docker-toolbox
+
+#. Create the virtual machine that Docker will run within::
+
+        docker-machine create --driver virtualbox --virtualbox-memory 2048 kuma
+
+#. Configure the environment variables::
+
+        eval "$(docker-machine env kuma)"
+        export COMPOSE_FILE=`pwd`/docker/docker-compose.yml
+        export COMPOSE_PROJECT_NAME=kuma  # String prepended to every container.
```

Comment:
Can we somehow hardcode that? Or add to a .env file so we don't have to run these commands all the time?